pkg/scheduler/objects/application_state.go

```diff
@@ -143,6 +143,10 @@ func NewAppState() *fsm.FSM {
 			},
 			fmt.Sprintf("enter_%s", Starting.String()): func(event *fsm.Event) {
 				app := event.Args[0].(*Application) //nolint:errcheck
+				app.queue.incRunningApps()
+				app.queue.decAllocatingAcceptedApps(app)
+				metrics.GetQueueMetrics(app.queuePath).IncQueueApplicationsRunning()
```

Comment:
We have this call at two places now: `enter_Starting` and `enter_Running`.

Reply:
good catch, removed the one in enter_Starting, kept the one in enter_Running